pkgs/games/nottetris2/default.nix

```
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, zip, love_0_7, lua, makeWrapper, makeDesktopItem }:
+
+let
+  pname = "nottetris2";
+  version = "2.0";
+
+  desktopItem = makeDesktopItem {
+    name = "nottetris2";
+    exec = pname;
+    comment = "It's like tetris, but it's not";
+    desktopName = "nottetris2";
+    genericName = "nottetris2";
+    categories = "Game";
+  };
+
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "Stabyourself";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "17iabh6rr8jim70n96rbhif4xq02g2kppscm8l339yqx6mhb64hs";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ lua love_0_7 zip ];
```

```suggestion
  nativeBuildInputs = [ zip ];
  buildInputs = [ lua love_0_7 makeWrapper ];
```
makeWrapper works as a native build input, confusingly, but zip should be native here.